uqcsbot/scripts/weather.py

```diff
@@ -0,0 +1,214 @@
+from uqcsbot import bot, Command
+from urllib.request import urlopen
+import xml.etree.ElementTree as ET
+from datetime import datetime as DT
+
+
+def get_xml(state: str):
```

Comment:
Not 100% necessary, but would be nice if you included the return type in the typing.

Reply:
Added.